src/auth/services/auth.service.ts

```diff
@@ -52,9 +52,14 @@ export class AuthService {
         lifetime?: number,
     ): CookieOptions {
         return {
-            maxAge:
-                lifetime ||
-                Number(this.configService.get<string>('JWT_LIFETIME')) / 1000,
+            expires: lifetime
```

Comment:
suggestion(non-blocking): I think `lifetime` could be misunderstood here. I suggest `timeTolive` or something similar. Ignore this if `lifetime` is a convention or standard.

Reply:
if we change this, we should also change the .env variable name